locale/en/emails.po

```diff
@@ -454,3 +454,16 @@ msgstr ""
 "<p>Thank you for considering {$contextName} as a venue for your work.</p>"
 "<p>Kind regards,</p>"
 "{$contextSignature}"
+
+msgid "emails.changeProfileEmailInvitationNotify.subject"
+msgstr "Confirm account contact email change request"
+
+msgid "emails.changeProfileEmailInvitationNotify.body"
+msgstr ""
+"<p>Dear {$recipientName},</p>"
+"<p>You are receiving this email because we have requested a change of your email to {$newEmail}.</p>"
```

Comment:
"we" is likely to be confusing here. Suggest: "someone has requested a change of your email".

Reply:
Changed.